Sources/RSocketSwiftNIO/wss/WebSocketClient+Connect.swift

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2015-present the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import RSocketCore
+
+public extension WebSocketClientTransport {
+    func connect() -> DuplexConnection {
+        fatalError("connect method is not implemented yet")
+    }
+}
```

Comment:
Since this file contains the actual implementation of the protocol, could we move the protocol conformance to this file and remove it from the base definition?
```suggestion
public extension WebSocketClientTransport: ClientTransport {
    func connect() -> DuplexConnection {
        fatalError("connect method is not implemented yet")
    }
}
```

Reply:
Yes, I did add them to one file and this will be part of the next PR